src/MetaMorpho.sol

```diff
@@ -759,10 +778,6 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
             Id id = withdrawQueue[i];
             MarketParams memory marketParams = _marketParams(id);
 
-            // The vault withdrawing from Morpho cannot fail because:
-            // 1. oracle.price() is never called (the vault doesn't borrow)
-            // 2. `_withdrawable` caps to the liquidity available on Morpho
-            // 3. virtually accruing interest didn't fail in `_withdrawable`
```

Comment:
Why is this removed? This comment is still correct

Reply:
I think it's not correct.

> // 3. virtually accruing interest didn't fail in `_withdrawable`

`_withdrawable` can fail because of the IRM.

Reply:
Yes, but that's not what is stated in this comment
The comment states that withdrawing from Morpho cannot fail, given the 3 assumptions
Perhaps you want to rephrase it

Reply:
I don't get the point of this comment in fact. `_simulateWithdrawMorpho` is used in `_maxWithdraw` in turn used in `maxWithdraw` and `maxRedeem`.

So there's no withdraw involved.

Maybe we want to say the following:

> If `_simulateWithdrawMorpho` did not fail then a withdraw on Morpho won't fail.

It's true only atomically though.

Or am I missing something?

Reply:
The point of the comment is to justify that `_simulateWithdrawMorpho` accounts for what's expected, ie what will actually be accounted for by `_withdrawMorpho`

In `_withdrawMorpho` we try/catch so there may be a hidden revert in `_withdrawMorpho` that may not be accounted for in `_simulateWithdrawMorpho`
So the comment actually justifies that this case doesn't exist, given the 3 assumptions listed

Reply:
To me the comment as it is now is unclear. And maybe we should move the comment in the Natspecs instead

Reply:
```solidity
// Withdrawing from Morpho cannot fail in `_withdrawMorpho` because:
// 1. oracle.price() is never called (the vault doesn't borrow)
// 2. virtually accruing interest didn't fail in `_withdrawable`
// 3. `_withdrawable` caps to the liquidity available on Morpho
```

Reply:
And if you want update the natspec of `_simulateWithdrawMorpho` to specify that it replicates the behavior of `_withdrawMorpho` and should account for the same things